website/pages/api-docs/secret/databases/index.mdx

```diff
@@ -48,19 +48,48 @@ list of additional parameters.
   executed to rotate the root user's credentials. See the plugin's API page for more
   information on support and formatting for this parameter.
 
+~> It is recommended that you use a super-user rather than the root user when
```

Comment:
I don't think we should suggest super-user.  Really all you need is a role that can `CREATE`, `UPDATE` and `DELETE` users.  Super-user has a lot more privileges that are unnecessary.

Reply:
:+1: